### DISCUSSION TITLE
Using SimpleSDXL with low VRAM

### DISCUSSION THREAD

**Discussion by tmprabubiz** [metercai/SimpleSDXL#64] [Q&A]
SimpleSDXL is an excellent development, I appreciate.

Please try  to help the users with low VRAM to experiment SimpleSDXL by implementing BitsandBytes kind of method as in FORGE.

Thank You

**Comment by DavidDragonsage** [comment]
Referenced in  **[Bug]: Produces Black Images Only #61**

**Comment by metercai** [comment]
SimpleSDXL2 already supports the Flux models of bnb-nf4, bnb-nf4-v2, and gguf Qx series. Some users have successfully started running in the 4G VRM environment.

**Reply by DavidDragonsage** [reply]
That is good to know! 

Yes, the problem with my 4GB video card may that is only a GTX 1650. That combination was never officially supported in mainline Fooocus. Fortunately I will be replacing it soon.

**Comment by DavidDragonsage** [comment]
My case the issue with the 4GB card has effectively been resolved.

I would suggest that this item be closed also, unless someone comes forward with the case of 4GB VRAM card that was _actually approved_ for running mainline Fooocus but cannot run SimpleSDXL. That would only apply to a Nvidia RTX 2XXX or higher graphics card with 4GB VRAM - _not_ the Nvidia GTX 1XXX that I had.
